core/src/it/scala/org/ensime/indexer/SearchServiceSpec.scala

```diff
@@ -328,4 +400,9 @@ object SearchServiceTestUtils {
     case cdef: ClassDef => Set(cdef)
     case TypeHierarchy(cdef, typeRefs) => Set(cdef) ++ typeRefs.flatMap(hierarchyToSet)
   }
+
+  def findUsages(fqn: String)(implicit service: SearchService): List[FqnSymbol] = Await.result(service.findUsages(fqn), Duration.Inf).toList
+
+  // 2.10 scalap has slightly different formatting for method names
+  def unifyMethodName(s: String): String = s.replaceAll(" : ", ": ")
```

Comment:
maybe heavyweight, but this could go into the `scala-2.10` source directory and then implement it per-scala version.